Introductory Tests/udp_class.py

```diff
@@ -0,0 +1,93 @@
+# General
+import numpy as np
+
+# For computing acceleration and potential
+import polyhedral_gravity as model
+
+# For working with the mesh
+import mesh_utility
+
+# For Plotting
+import pyvista as pv
+
+class udp_obj:
```

Comment:
I would suggest to name files and classes the same to keep imports easy. Then you can just

`from udp import udp` and don't have to think about the name

Further, I would rename this maybe into udp_initial_condition or something like this indicate this is the one where you optimize initial condition? (later ones be called differently?